kalite/coachreports/templates/coachreports/timeline_view.html

```diff
@@ -50,4 +51,4 @@
             TimelineplotTopics(null);
         });
     </script>
-{% endblock headjs %}
\ No newline at end of file
+{% endblock headjs %}
```

Comment:
I did not change anything here....i dont why why this is showing i didnt even press enter.. should i recommit or is it fine?